models/articles.models.js

```diff
@@ -2,10 +2,12 @@ const db = require("../db/connection")
 
 exports.selectArticle = (id) => {
   let queryStr = "SELECT * FROM articles "
+  const queryParams = []
```

Comment:
There is no need for this to be here. You are absolutely ok to write an array with the id after the query.